c++/src/H5AbstractDs.h

```diff
@@ -68,7 +68,7 @@ class H5_DLLCPP AbstractDs {
     virtual H5std_string fromClass() const = 0;
 
     // Destructor
-    virtual ~AbstractDs();
+    virtual ~AbstractDs() = default;
```

Comment:
For a `virtual` destructor, prefer putting this in the `.cpp` file; i.e.,:
```c++
AbstractDs::AbstractDs() = default;
```
That way, you don't end up generating a definition in every translation unit that includes the header.

Reply:
I have never seen a codebase do that... I can't find much discussion about it either, except for [this](https://stackoverflow.com/questions/58774272/virtual-default-destructor-definition-in-source-file-vs-in-header-file), where the answers are not so conclusive to me...

I'd like to RTFM on this if you could point me somewhere...